R/Analyze_Fisher.R

```diff
@@ -24,9 +24,9 @@
 #'
 #' @param dfTransformed `data.frame` Transformed data for analysis. Data should
 #'   have one record per site with expected columns: `GroupID`, `GroupLevel`,
-#'   `Numerator`, `Denominator`, and `Metric`. For more details see the Data
-#'   Model vignette: `vignette("DataModel", package = "gsm.core")`. For this
-#'   function, `dfTransformed` should typically be created using
+#'   `Numerator`, `Denominator`, and `Metric`. For more details see the
+#'   \href{https://gilead-biostats.github.io/gsm.core/articles/DataModel.html}{Data Model Vignette}.
```

Comment:
```suggestion
#'   [Data Model article](https://gilead-biostats.github.io/gsm.core/articles/DataModel.html).
```
We can use markdown in help. That will make it easier to update it when we change it to openrbqm, for example. I don't want to call it a vignette in case a CRAN reviewer freaks out about it.

Reply:
yeah- partway through i switched to this format. weird old latex habit crept in here for some reason 🙃